rootfs-builder/clearlinux/config.sh

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+#Use "latest" to always pull the last Clear Linux Release
+OS_VERSION=latest
```

Comment:
It might be better to specify this as:

```
OS_VERSION=${OS_VERSION:-latest}
```

... to allow an explicit version to be used?